File: Data Structures & Algorithms/reverse-bits/submission-1.java
class Solution {
    public int reverseBits(int n) {
        int ans = 0;
        int power = 31;
        for(int i=0;i<32;i++){
            int bit = n&1;
            System.out.print(bit + " ");
            if(bit==1){
                ans += (1<<power);
            }
            n = n>>1;
            power--;
        }
        return ans;

    }
}

File: Data Structures & Algorithms/merge-strings-alternately/submission-1.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string ans;
        int l1 = 0;
        int l2 = 0;
        int n1 = word1.size();
        int n2 = word2.size();

        while(l1<n1 && l2 < n2){
            ans.push_back(word1[l1]);
            ans.push_back(word2[l2]);
            l1++;
            l2++;
        }
        ans += word1.substr(l1,n1);
        ans += word2.substr(l2,n2);
        return ans;

    }
};